frontend/src/app/organization/widgets/organization-details-info-card/organization-details-info-card.widget.html

```diff
@@ -25,46 +25,46 @@
         <!-- Organization Name -->
         <div class="organization-name-section">
           <mat-card-title class="user-name">
-            {{ organization!.name }}
+            {{ organization.name }}
           </mat-card-title>
         </div>
       </div>
 
       <!-- Organization Description -->
-      <p
-        class="organization-description"
-        *ngIf="organization!.long_description; else short_description">
-        {{ organization!.long_description }}
+      <p class="organization-description">
+        {{
+          organization.long_description
+            ? organization.long_description
+            : organization.short_description
```

Comment:
I believe this could also be done using short circuiting:

`organization.long_description || organization.short_description`

JavaScript evals the `||` to the first truthy value and does not coalesce to `true`

Reply:
Great point!